java/ingest/src/main/java/sleeper/ingest/IngestRecords.java

```diff
@@ -36,24 +34,10 @@ public class IngestRecords {
 
     private final IngestCoordinator<Record> ingestCoordinator;
```

Comment:
Maybe we should make IngestRecords AutoCloseable so we can put it in a try-with-resources block to ensure the IngestCoordinator is closed?

Reply:
All usages of IngestRecords already call closeWithReturningList() on the IngestCoordinator, and are expecting an IngestResult at the end, I'm not sure how we could implement AutoCloseable with this class